README.asciidoc

```diff
@@ -437,7 +437,8 @@ and
 <1> Any headers in the appendix or in the preface start
     out-of-sequence at `level 2`, not at `level 1`.
 
-[sect3]
+ifdef::asciidoctor[[section]]
+ifndef::asciidoctor[[sect3]]
```

Comment:
It is rare to have a section called `Glossary` that isn't actually the glossary. So AsciiDoc has special syntax to make it into a normal section (`[sect3]`). Asciidoctor also has special syntax, but it isn't the *same* special syntax.